File: DrinkerAPI/Models/Coctail.cs
using System.Collections.Generic;

namespace DrinkerAPI.Models
{
    public class Coctail
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Category { get; set; }
        public string Alcoholic { get; set; }
        public string Glass { get; set; }
        public string Instructions { get; set; }
        public string PhotoUrl { get; set; }
        public string DateModified { get; set; }
        public virtual IList<Ingredient> Ingradients { get; set; }
        public int? UserId { get; set; }
        public bool IsAccepted{get;set;}
        public virtual ICollection<FavouriteCoctail> FavouritedByUsers { get; set; }
    }
}

File: drinkerClient/src/app/_models/coctail.ts
import { Ingredient } from "./ingredient";

export interface Coctail {
    id: number;
    name: string;
    category: string;
    alcoholic: string;
    glass: string;
    instructions: string;
    photoUrl: string;
    dateModified: Date,
    isAccepted: boolean,
    ingradients: Ingredient[]
}